src/core/setup/index.ts

```diff
@@ -66,7 +66,7 @@ export function createTest(options: Partial<TestOptions>): TestHooks {
     beforeEach,
     afterEach,
     afterAll,
-    setup,
+    beforeAll,
```

Comment:
to avoid a breaking change we could also return setup but mark it as deprecated

Reply:
I have added `setup` to the return and marked it deprecated